.github/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: Make pcengines/apu2 kernel
```

Comment:
This is very nice but will be painful to scale as we add mainboards?

But it does anchor some nice constants: 
mainboard path structure
flashkernel as a name all boards should offer
so I'm inclined to say yes? wdyt?

Reply:
Yes, I think we'll want a `make all` sooner than later.